libs/shared/ui/src/lib/components/block-content-delete/block-content-delete.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { screen, render } from '__tests__/utils/setup-jest'
+import BlockContentDelete, { BlockContentDeleteProps } from './block-content-delete'
+
+describe('BlockContentDelete', () => {
+  const props: BlockContentDeleteProps = {
+    title: 'Delete',
+    description: 'my-description',
+    callback: jest.fn(),
+  }
+
+  it('should render successfully', () => {
+    const { baseElement } = render(<BlockContentDelete {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should have custom icon', () => {
+    props.list = [
+      {
+        text: 'Databases',
+      },
+      {
+        text: 'Applications',
+        icon: 'icon-solid-chart-area',
+      },
+    ]
+
+    render(<BlockContentDelete {...props} />)
+
+    const firstElement = screen.getByTestId(props.list[0].text)
+    const secondElement = screen.getByTestId(props.list[1].text)
+
+    expect(firstElement.querySelector('.icon-solid-trash')?.classList.contains('icon-solid-trash')).toBe(true)
+    expect(secondElement.querySelector('.icon-solid-chart-area')?.classList.contains('icon-solid-chart-area')).toBe(
+      true
+    )
```

Comment:
Need to add a test that simulate the click on the button to see if the callback has been called. 
act() click spy.hasbeencalled